apps/gateway/src/chat/chat.ts

```
@@ -942,11 +942,39 @@
+								prompt_tokens: data.usageMetadata.promptTokenCount > 0 
+									? data.usageMetadata.promptTokenCount 
+									: (() => {
+										// Calculate prompt tokens if missing or 0
+										try {
+											const chatMessages: ChatMessage[] = messages.map((m) => ({
+												role: m.role,
+												content: typeof m.content === "string" ? m.content : JSON.stringify(m.content),
+												name: m.name,
+											}));
+											return encodeChat(chatMessages, DEFAULT_TOKENIZER_MODEL).length;
+										} catch (error) {
```

ESLint

[error] 'error' is defined but never used. Allowed unused caught errors must match /^_/u.

(no-unused-vars)